### DISCUSSION TITLE
Measuring the cortical thickness of the region stimulated by TMS

### DISCUSSION THREAD

**Discussion by FannyTH** [simnibs/simnibs#151] [9. Results and analysis]
Hi SimNIBS experts,
Is it possible to measure the cortical thickness of the region stimulated by TMS from the mesh generated by SimNIBS? If so, how?
Thank you in advance

**Comment by oulap** [comment]
The surface generation tool (CAT12) does put out a thickness estimate. This is stored in the surface-folder (inside the m2m-folder) and you can visualize it in Freeview by first opening the surface file, e.g., lh.central.gii, and then opening the lh.thickness file as an overlay. 

However, we have not validated the thickness estimates at all, so please do keep this in mind. Estimates from freesurfer would likely be somewhat different.

If you still want to use the thickness, you can take the e-field values mapped to the middle cortical surface, threshold those at a given electric field strength, and then read the corresponding thickness estimates from the surface nodes that fall within the e-field threshold.

**Comment by dimst23** [comment]
Hi!
Just to avoid opening a new discussion topic given that my question is directly related to this one, I am asking here.

Is there a way to calculate the cortical thickness from a `charm` segmented model? I have the freesurfer model as well, but I want to make sure that the characteristics from the charm model are similar to freesurfer.

Thank you!

**Reply by oulap** [reply]
charm allows you to pass the freesurfer surfaces into the head model generation. That way you can be sure that the cortical segmentation is very similar to the FS one. The flag to use is: --fs-dir

**Reply by dimst23** [reply]
My question is related more to #316 and because it is whole question by itself, I opened that to avoid flooding this thread with irrelevant content.

I used the `--fs-dir` flag.
